src/System.Text.Json/tests/Serialization/Array.WriteTests.cs

```diff
@@ -49,13 +49,19 @@ public static void WriteByteArray()
             Assert.Equal($"\"{Convert.ToBase64String(input)}\"", json);
         }
 
-        [Fact]
-        public static void WriteTwo2dByteArray()
+        [Theory]
+        [InlineData(true, @"[
+  ""{0}"",
+  ""{0}""
+]")]
+        [InlineData(false, @"[""{0}"",""{0}""]")]
+        public static void WriteTwo2dByteArray(bool formatted, string format)
```

Comment:
I am surprised this is improving test coverage. If so, can we hit this scenario with a unit test directly testing `Utf8JsonWriter`?

nit: None of the tests here test the `WriteIndented = true` JsonSerializerOption (or any options customization). Consider keeping the existing test as is, but instead add a new test which does so.

Maybe: `WriteTwo2dByteArrray_CustomOptions`

Reply:
It covers the if statement in screenshot below. I figured I'd edit one of the tests that go through the function already.
Following your suggestion I'm going to revert and leave this one as it is and make changes to `WritingTooLargeBase64Bytes` test from src/System.Text.Json/tests/Utf8JsonWriterTests.cs file for this purpose instead.

![image](https://user-images.githubusercontent.com/1119735/61851319-e32a6c80-aeb6-11e9-9cd6-9983b47b2c47.png)

Reply:
> If so, can we hit this scenario with a unit test directly testing `Utf8JsonWriter`?

 Following your suggestion I'm going to revert and leave this one as it is

To clarify, I meant adding the test case to Utf8JsonWriter **in addition to** testing the same code path via the serializer. I can no longer see older commits, but I think the original test still had value.

As an FYI, in general, it is easier during review to keep the history of the commits (rather than squashing/force pushing). During merge, we squash the commits anyway, but it is useful to do incremental review of just the latest changes rather than having to review the whole PR again. However, this PR is small enough that it isn't a concern.